Rebalances a few guns strange sundering values

ClosetedSkeletonGH opened the pull request:
## About The Pull Request
AS OF 2023/01/01 15:05 EST THIS PR HAS HAD ITS VALUES CHANGED, CHECK THE LIST

This PR changes the sundering values of some guns that frankly, made very little sense.

Flechette: 17 -> 14 ( 7 + 5 + 5 -> 6 + 4 + 4)
88Mod auto pistol: 2 -> 1.25 
Martini Henry: 10 -> 15
Repeater: 1.25 -> 3.5
DMR: 2 -> 3.5

## Why It's Good For The Game
Some balancing of the sunder was overall very fucky. I tried fixing up some of the stranger cases now that sunder is being examined more closely. Im a data dork and have charts if anyone wants to see comparisons, @ me on #balance

1 shot of 88mod was 4 times more sundering than a basic assault rifle
Martini Henry, a gun known for being extremely powerful/an elephant hunter is no longer worth 20 MP-19 shots
Repeater was worth the same amount as an AR-11, now it is worth 4 AR-11 shots
DMR, for being a DMR, didnt feel like only 2 sundering felt appropriate


## Changelog
:cl:
balance: flechette shells sundering changed from: 17 -> 14 ( 7+5+5 -> 6+4+4 )
balance: 88Mod auto pistol sundering changed: 2 -> 1.25 
balance: Martini Henry sundering changed: 10 -> 15
balance: Repeater sundering changed: 1.25 -> 3.5
balance: DMR sundering changed: 2 -> 3.5
/:cl:

ClosetedSkeletonGH commented:
Before salty nerds come whining:

88Mod
![image](https://user-images.githubusercontent.com/120620754/210156064-815f8ac0-da60-47ac-b628-82f90de20dc4.png)

Mosin + SR-127
![image](https://user-images.githubusercontent.com/120620754/210156081-f24e6c7e-c954-40b5-a7bd-6ea8bd1dd4d7.png)

Martini Henry
![image](https://user-images.githubusercontent.com/120620754/210156086-307dd1ff-0e83-46ba-839d-8477c3d94f1b.png)

Repeater
![image](https://user-images.githubusercontent.com/120620754/210156102-fb6064a9-9891-4f76-b46f-4c88e6cca62e.png)

![image](https://user-images.githubusercontent.com/120620754/210156110-fa00f166-4fef-4f0a-81e7-599ec177963f.png)

And before someone says, yes, I know this is not the best way of measuring sunder, but the important part is to compare before and after on the amount of sundering done at certian %'s of sunder remaining. Sunder isnt about a single mag dump, its about a single xeno going through repeated battles.

Losenis commented:
5 SUNDER FOR THE STAGGERLOCK GUN FUCKING WEW

ClosetedSkeletonGH commented:
> 5 SUNDER FOR THE STAGGERLOCK GUN FUCKING WEW

If you cant counter a Repeater thats a skill issue

Pariah919 commented:
>balance: 88Mod auto pistol sundering changed: 2 -> 1.25
balance: SR-127 & Mosin sundering changed: 15 -> 12.5
balance: Martini Henry sundering changed: 10 -> 15
balance: Repeater sundering changed: 1.25 -> 5
balance: DMR sundering changed: 2 -> 3.5
>
why? 88mod is fine, having good sunder and AP is the gimmick behind it (beyond burst fire.)
I haven't seen a TL in so long
martini is a meme so this is okay i guess
repeater is fine
i'd do 2.5 or 3 before 3.5

Al-1ce commented:
>balance: 88Mod auto pistol sundering changed: 2 -> 1.25


Sure.

>balance: SR-127 & Mosin sundering changed: 15 -> 12.5


**Absolutely not.** The Mosin is a meme weapon: the TL-127 is a specialized sniper rifle. It's not always easy to use and requires a lot of patience to be efficient.

Let me be clearer here; I don't have any qualms with the mosin being nerfed, despite it's already numerous drawbacks. But touching the sniper (my baby) is a big fucky-wucky and an absolute NO on my part. I'm sure many others can agree.

>balance: Martini Henry sundering changed: 10 -> 15


Sure. It's not exactly consistent to nerf one meme-gun and buff another but I haven't used either of these in ages so do w/e.

>balance: Repeater sundering changed: 1.25 -> 5


This _may_ be a bit much. Stagger on xenos is already difficult to play against, but having additional sunder could really elevate the repeater to req-item status. 

I'd like to see this played around with for a more fluid perspective all-the-same. Sure, I guess.

balance: DMR sundering changed: 2 -> 3.5


I doubt this will change much in terms of balance. Sure.

Naaanii commented:
Repeater is already good, don't care much for the others.

ClosetedSkeletonGH commented:
Reverted the sniper change per feedback, also added in flech because wow 17 sunder for something that can have 4 shots put out in a second (53 sunder total in current equation if every shot lands)

Coastfront commented:
rest in piece 88mod autopistol they finally found you my boy
